src/MongoDB.Driver/MongoClient.cs

```diff
@@ -213,6 +213,14 @@ public sealed override IMongoDatabase GetDatabase(string name, MongoDatabaseSett
             return UsingImplicitSession(session => ListDatabaseNames(session, cancellationToken), cancellationToken);
         }
 
+        /// <inheritdoc />
```

Comment:
Change the previous overload to *not* do any work. Overloads that exist only to provide default values for optional parameters should delegate to another overload passing the default value:

    public sealed override IAsyncCursor<string> ListDatabaseNames(
        CancellationToken cancellationToken = default(CancellationToken))
    {
        return ListDatabaseNames(options: null, cancellationToken);
    }

Same for corresponding async method.

Reply:
Done.